packages/tools/pie-icons-configs/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "description": "",
   "exports": {
-    ".": "./configs.js",
+    "./configs": "./configs.js",
```

Comment:
Does this change mean that you'd need to import `@justeattakeaway/pie-icons-configs/configs/` rather than just `@justeattakeaway/pie-icons-configs`?

I think it makes more sense as `.` (without the `/configs`) – mainly as not many package don't have a default export, but interested to hear other thoughts potentially.

Reply:
Might be worth repeating what we do in the [eslint config package](https://github.com/justeattakeaway/pie/blob/3865b0026261d4fe6db836f7dc57c85aa13e4c76/packages/tools/eslint-config-pie/package.json#L5C3-L9C5)

Reply:
ah yep I was getting an error earlier to do with the export so changed it to `./configs`, but @dandel10n helped me solve this this morning so I'll revert the change 👍